boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -62,7 +62,7 @@ CONVENTION_DIR := boilerplate/openshift/golang-osd-operator
 # https://www.gnu.org/software/make/manual/make.html#index-_002eDEFAULT_005fGOAL-_0028define-default-goal_0029
 .DEFAULT_GOAL :=
 .PHONY: default
-default: go-build
+default: go-check go-test go-build
```

Comment:
I'm no `make` expert but the general workflow I have seen is that simply running `make` just builds the binary/artifact. OTH running `make test` runs checks and other verification. So do we want to keep that workflow here as well. In other words should default just be `go-build`.

Reply:
I did actually ask about this a couple weeks ago, because I thought the same thing from my earliest experiences with `make`. I was told this was typical. Thinking of it empirically, IMO it makes sense for the default to be the case I would most commonly/frequently want to invoke as a developer, and this seems reasonable.

Anyway, happy to discuss further. This PR isn't going anywhere for a little while -- I'd like to hold it until we can reinstate a way to run check & test in the container.